citi-onboarding/Delta-odonto
# Hotfix/our clients

## What I did
#### Several areas needed small adjustments. The following areas received these changes:

1. Adjustments on title pattern sizes for mobile version

2. Adjustments on 'services section' (Replacing position absolute with flexbox-order)

3. Adjustments on sidebar links (Not working before)

4. Adjustments on contact button (It was floating around the page as the user scroll it)

5. Adjustments on 'our clients' section (Replacing position absolute with flexbox order)

## How to test
- On terminal you should type de following commands to access the server folder and run the index file:
```
cd server
node index.js
```
- Open a second terminal on client folder and run the following commands to habilite the Keystone interface
```
cd client
yarn server
```

- Go to a browser and access the adress http://localhost:3000 and you should be able to see the landing page.

- You should be able to interact with all components and check if it is according with the mockup.
